packages/fxa-settings/src/pages/Signin/utils.ts

```diff
@@ -327,7 +321,8 @@ export async function handleNavigation(navigationOptions: NavigationOptions) {
   }
 
   // Note that OAuth redirect can only be obtained when the session is verified,
-  // otherwise oauth/authorization endpoint throws an "unconfirmed session" error
+  // otherwise oauth/authorization endpoint throws an "unconfirmed session" error.
+  // The only exception to this is
```

Comment:
Ah oops, need to update this comment to something like, "The only exception to this is the type C unverified session noted above"